mkdocs.yml

```diff
@@ -97,10 +97,12 @@ nav:
           - Overview: ui/overview.md
           - AG-UI: ui/ag-ui.md
           - Vercel AI: ui/vercel-ai.md
+          - Web Chat UI: ui/web.md
```

Comment:
To the user, this is a Pydantic AI feature, not a UI event stream integration (even if it technically uses that functionality), so it shouldn't be under this section. Instead, let's make it its own dedicated page under `Documentation` above. We can just move it to `web.md` as well, no `ui/` needed.